template/Dockerfile

```diff
@@ -52,6 +52,17 @@ RUN micromamba install -y --name base --file /tmp/$ENV_IN_FILENAME && \
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
 RUN sudo ln -s $(which python3) /usr/bin/python
 
+# Install glue kernels, and move to shared directory
+RUN install-glue-kernels && \
+    jupyter-kernelspec install /opt/conda/lib/python3.10/site-packages/aws_glue_interactive_sessions_kernel/glue_pyspark --user && \
```

Comment:
We shouldn't hardcode a python version in this path from the template dockerfile, as that can change depending on the sagemaker-distribution major version. Is there an env variable or something we can use to dynamically resolve the path?

Reply:
I think there's a similar use case in the analytics PR. Might be able to use same env var used there: https://github.com/aws/sagemaker-distribution/pull/90/files

Reply:
Added dynamic site packages logic